src/assets/i18n/en.json

```diff
@@ -0,0 +1,226 @@
+{
+  "abnormalities":"Abnormalities",
+  "abnormalityDelete":"Delete abnormality",
+  "abnormalityDeleteQuestion":"Do you really want to delete this abnormality ?",
+  "abnormalityDetail":"Abnormality details",
+  "abnormalityLeaveMessage":"The abnormality contains some validation errors and have not been saved. Do you still want to go to the building abnormalities list ?",
```

Comment:
"...and HAS not been saved..."

De même qu'à toutes les autres phrases semblables dans le reste du document.